src/test/java/posters/cucumber/support/RegisterSupport.java

```diff
@@ -27,24 +22,27 @@ public RegisterSupport(GlobalStorage storage)
     @Given("^user setup: \"([^\"]*)\", \"([^\"]*)\", \"([^\"]*)\", \"([^\"]*)\"$")
     public void setUpUser(String firstName, String lastName, String eMail, String password)
     {
-        // set up user for the clean up steps
+        // create a user for later use (e.g. register or delete)
         storage.user = new User(firstName, lastName, eMail, password);
     };
 
     @Given("^login page is opened after registration$")
     public void registerUserSetup()
     {
         // use the user coming from dependency injection
-        registerUser(storage.user);
+        // registerUser(storage.user);
```

Comment:
@oomelianchuk BTW: what happened to this `registerUser(storage.user);` function. Please utilize it or remove it and the comment if it isn't needed anymore.